lib/workers/pr/changelog/release-notes.ts

```diff
@@ -179,14 +179,15 @@ export async function getReleaseNotesMd(
     if (changelogParsed.length >= 2) {
       for (const section of changelogParsed) {
         try {
-          const [heading] = section.split('\n');
+          const [heading] = section.replace(/[[\]()]/g, ' ').split('\n');
           const title = heading
             .replace(/^\s*#*\s*/, '')
             .split(' ')
             .filter(Boolean);
           let body = section.replace(/.*?\n(-{3,}\n)?/, '').trim();
           for (const word of title) {
-            if (word.includes(version)) {
+            const isUrl = /^(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[a-z0-9]+([-.]{1}[a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?$/gm;
```

Comment:
Is there may be a better way to approach this? e.g. can we simply use URL.parse and see if it succeeds?

Reply:
let me check that, and will correct it.